### PR TITLE
Move feedback component out of components.ko.js

### PR DESCRIPTION
**Pull request by orangejenny**
## Technical Summary
As discussed here: https://github.com/dimagi/commcare-hq/pull/35166#discussion_r1786427040

This PR just updates the feedback component, which is the least-frequently-used one. Since comments are indeed unreliable, I grepped for `<feedback` to find  the two places it's used (OData feeds and the Explore Case Data report), updated those, and tested that the widget still works.

Bonus, this PR makes it so that `components.ko.js` no longer needs to be split into a B3 and B5 version, which will reduce the number of split js files at least a little bit. I'll follow up on that in a different PR.

### Safety story
Smoke tested locally.

### Automated test coverage

unlikely

### QA Plan

not requesting QA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by orangejenny** [comment]
Ignoring lint errors, which are in export js files and I'm guessing are fixed in the export B5 migration.

File: corehq/apps/hqwebapp/static/hqwebapp/js/components/bootstrap3/feedback.js [hunk 1]
```diff
@@ -74,4 +76,17 @@ hqDefine('hqwebapp/js/components/bootstrap3/feedback', [
         },
         template: '<div data-bind="template: { name: \'ko-feedback-template\' }"></div>',
     };
+
+    ko.components.register('feedback', component);
+
+    $(function () {
+        _.each($('feedback'), function (el) {
+            var $el = $(el);
+            if (!($el.data('apply-bindings') === false)) {
```
Comment (millerdev) [inline corehq/apps/hqwebapp/static/hqwebapp/js/components/bootstrap3/feedback.js:85]: nit: is there a reason the `!==` operator was not used here?
```suggestion
            if ($el.data('apply-bindings') !== false) {
```

File: corehq/apps/hqwebapp/static/hqwebapp/js/components/bootstrap5/feedback.js [hunk 2]
```diff
@@ -69,4 +71,17 @@ hqDefine('hqwebapp/js/components/bootstrap5/feedback', [
         },
         template: '<div data-bind="template: { name: \'ko-feedback-template\' }"></div>',
     };
+
+    ko.components.register('feedback', component);
+
+    $(function () {
+        _.each($('feedback'), function (el) {
+            var $el = $(el);
+            if (!($el.data('apply-bindings') === false)) {
```
Comment (millerdev) [inline corehq/apps/hqwebapp/static/hqwebapp/js/components/bootstrap5/feedback.js:80]: nit
```suggestion
            if ($el.data('apply-bindings') !== false) {
```

**Review by millerdev — APPROVED** [review]

File: corehq/apps/hqwebapp/static/hqwebapp/js/components/bootstrap3/feedback.js (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/hqwebapp/static/hqwebapp/js/components/bootstrap3/feedback.js:85]: nope, that's a good update

**Review by millerdev — APPROVED** [review]

**Pull request merged**